UPGRADE.md

```diff
@@ -0,0 +1,114 @@
+# Upgrade to Ceramic One from Kubo
+
+This document describes how to upgrade an existing Ceramic node using Kubo to run with `ceramic-one` while preserving existing data.
+
+## Overview
+
+Follow these steps for basic upgrade of a Ceramic node with some downtime.
+
+1. Shutdown existing `js-ceramic` node.
+2. Migrate data from IPFS blocks into Sqlite database file `ceramic-one` can consume.
+3. Start `ceramic-one` daemon.
+4. Configure `js-ceramic` to run against `ceramic-one`.
+5. Start `js-ceramic` node and validate.
+
+If you require a zero downtime upgrade see the section below that expands the basic steps for a near zero downtime upgrade.
+
+### Shutdown Existing Node
+
+First shutdown both the `js-ceramic` and `kubo` processes using whatever mechanism your infrastructure dictates.
+
+### Migrate IPFS Blocks to Ceramic Events
+
+Kubo stores data as IPFS blocks on disk.
+If you are not using the disk backed Kubo storage first migrate data to an on disk store following Kubo's instructions.
+
+These IPFS blocks are the raw pieces of Ceramic Events.
+In order to preserve this data we need to migrate them into Ceramic Events stored in as a Sqlite database file.
+
+The `ceramic-one` command contains a subcommand for migrating data from IPFS blocks.
+Install the `ceramic-one` binary following the instructions in README.md.
+
+See the complete usage information for the migration command:
+
+    ceramic-one migrations from-ipfs --help
+
+The command requires an input directory where the IPFS blocks are found and where the output storage directory.
+
+The following command will migrate data assuming the default directories for both Kubo and `ceramic-one`.
+Adjust the command arguments as needed by your infrastructure.
+
+    ceramic-one migrations from-ipfs -i ~/.ipfs/blocks -o ~/.ceramic-one --network mainnet
+
+This command will migrate all blocks stored in the `~/.ipfs/blocks` directory into the database file `~/.ceramic-one/db.sqlite3` for the `mainnet` network.
```

Comment:
I wonder if there is a step or an output that devs could use as a checkpoint to verify that this step in the migration process was completed correctly? If there is, ti would be worth including it here.

Reply:
Starting the js-ceramic process will fail if events haven't been migrated. I'll call that out here